cmd/sonobuoy/app/status.go

```diff
@@ -161,23 +206,6 @@ func printSummary(w io.Writer, status *aggregation.Status) error {
 	return nil
 }
 
-type pluginSummaries []pluginSummary
```

Comment:
FYI: moved this type data to the top of the page; somewhat of a nit/preference but I have always understood types to be defined at the top of the file under constants/variables.